pootle/apps/pootle_statistics/migrations/0019_convert_accept_sugg_subs.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-01 08:04
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import F
+
+
+WEB_FLAG = 1
+SUGG_ACCEPT_FLAG = 3
+
+
+def _assoc_state_accept_suggestion_subs(apps):
+    subs = apps.get_model("pootle_statistics.Submission").objects.all()
+    suggestions = apps.get_model("pootle_store.Suggestion").objects
+    accepted = suggestions.filter(state__name="accepted").values(
+        "review_time", "pk", "unit_id", "reviewer")
+    state_acc = subs.filter(type=SUGG_ACCEPT_FLAG).filter(field=3)
+
+    for suggestion in accepted.iterator():
+        suggestion["submitter"] = suggestion["reviewer"]
+        suggestion["creation_time"] = suggestion["review_time"]
+        suggestion_id = suggestion["pk"]
+        del suggestion["review_time"]
+        del suggestion["reviewer"]
+        del suggestion["pk"]
```

Comment:
I raised this because I was worried about what else can be in `suggestion`, thus my ask to make it explicit what we pass below. Anyway I notice now that it is used for filtering so it looks less concerning to me. I am happy to let it go.